app/javascript/packs/document-capture.tsx

```diff
@@ -120,6 +128,7 @@ const App = composeComponents(
         inPersonOutageMessageEnabled: inPersonOutageMessageEnabled === 'true',
         inPersonOutageExpectedUpdateDate,
         inPersonFullAddressEntryEnabled: inPersonFullAddressEntryEnabled === 'true',
+        optedInToInPersonProofing,
```

Comment:
do we need to typecast a string to a boolean like the lines above?
```suggestion
        optedInToInPersonProofing: optedInToInPersonProofing === 'true',
```

Reply:
yes i believe you're correct